int32[] axisInputs
int32[] buttonInputs
